Sources/Sentry/include/SentryTransactionContext+Private.h

```diff
@@ -37,6 +37,9 @@ SentryTransactionContext (Private)
                parentSampled:(SentrySampleDecision)parentSampled;
 
 #if SENTRY_TARGET_PROFILING_SUPPORTED
+
+@property (nonatomic, strong) SentryThread *threadInfo;
```

Comment:
Needed to expose this so we can set it in the new test case.

Reply:
`l`: Please add a comment that it's only required for testing, then.